### ISSUE TITLE
How to check if a space is duplicated from another one using HF API?

### ISSUE THREAD

**Issue by zhimin-z** [huggingface/hub-docs#1435]
I cannot find any related specifications in the documentation...Thanks!

**Comment by tfrere** [comment]
Hi! I'm working with @clefourrier - we're trying to automatically parse duplicated spaces (to ignore them),
using the **duplicated_from** key in **cardData**.

"**duplicated_from**" seems to need manual input by the space owner.
However, on the hub, there's a "Duplicated from" field which seems to automatically detect when a space was forked from another space.

Would there be a way to make this latter info available through the API, and/or to sync both methods?

**Comment by Wauplin** [comment]
Unfortunately I don't think this info is publicly available via API (cc @coyotte508)

**Comment by zhimin-z** [comment]
> Unfortunately I don't think this info is publicly available via API (cc [@coyotte508](https://github.com/coyotte508))

Thanks, but how could I automatically detect if a space is duplicated from other spaces?

**Comment by coyotte508** [comment]
yes I don't think so at the moment, although it's in DB (`originRepo`), so it's not too hard to add it to the `expand` fields

**Comment by coyotte508** [comment]
> > Unfortunately I don't think this info is publicly available via API (cc [@coyotte508](https://github.com/coyotte508))
> 
> Thanks, but how could I automatically detect if a space is duplicated from other spaces?

at the moment you can do:

```console
(curl https://huggingface.co/spaces/huggingfacejs/push-model-from-web | grep "Duplicated from") && echo "yes" || echo "no"
(curl https://huggingface.co/spaces/coyotte508/test-req | grep "Duplicated from") && echo "yes" || echo "no"
```
